src/gui/MainWindow.cpp

```diff
@@ -244,6 +246,7 @@ void MainWindow::setTouchscreenScrollingForDeviceMapping() {
             break;
         }
     }
```

Comment:
We might not need this block of code anymore with the updated touch settings, but I'm not 100% sure.

Reply:
Actually, the added line should go above this, since if we set a touchscreen to a different device class, there should be no kinetic scrolling regardless if touch draw is disabled or not.